dashboard/app/controllers/lti/v1/account_linking_controller.rb

```diff
@@ -0,0 +1,36 @@
+module Lti
+  module V1
+    class AccountLinkingController < ApplicationController
+      before_action :lti_account_linking_enabled?
+
+      # GET /lti/v1/account_linking/landing
+      def landing
+        render 'lti/v1/account_linking/landing_page'
+      end
+
+      # GET /lti/v1/account_linking/link_existing_account
+      def link_existing_account
```

Comment:
Please use a noun to name a GET action
```suggestion
      # GET /lti/v1/account_linking/existing_account
      def existing_account
```